lib/cluster.js

```diff
@@ -21,6 +21,6 @@
 
 'use strict';
 
-module.exports = ('NODE_UNIQUE_ID' in process.env) ?
-                  require('internal/cluster/child') :
-                  require('internal/cluster/master');
+module.exports = require(`internal/cluster/${
```

Comment:
I'm not particularly a fan of this multi-line inline replacement stuff. Can't we just set it to a variable and use the variable name instead?